core/src/main/java/org/mule/runtime/core/api/exception/Errors.java

```diff
@@ -151,7 +149,14 @@
     public static final String SOURCE_ERROR_RESPONSE_GENERATE_ERROR_IDENTIFIER = "SOURCE_ERROR_RESPONSE_GENERATE";
 
     /**
-     * Indicates that a severe error occurred. Cannot be handled. Other unhandleable errors should go under it.
+     * Indicates that an unknown and unexpected error occurred. Cannot be handled directly, only through ANY.
+     */
+    public static final String UNKNOWN_ERROR_IDENTIFIER = "UNKNOWN";
+
+    // UNHANDLEABLE
```

Comment:
Identifiers are rarely used alone. I'll add a distinction within the ComponentIdentifiers.